scripts/build-deps

```diff
@@ -44,6 +44,8 @@ echo ./configure
     --enable-debug=3 \
     --enable-gpl \
     --enable-libx264 \
+    --enable-libxml2 \
```

Comment:
Why these changes?

Reply:
I've simplified it in https://github.com/PyAV-Org/PyAV/pull/718/commits/cc5b261e80122e46853609bb01f718becf6da26e to just enable libxml2. The DASH demuxer, which is used in the tests, is not enabled if libxml2 is not enabled; libxml2 is used to read the DASH manifest XML file.

Reply:
Would a test which writes a sequence of PNG files also make sense?

Reply:
I think that would be worth doing (added in https://github.com/PyAV-Org/PyAV/pull/718/commits/d4b23725cb18303bdf809730156889d2c452e92b) and it actually exposed a bug that the DASH muxer hadn't (fixed in https://github.com/PyAV-Org/PyAV/pull/718/commits/f77e71464a259a72fb199fa76dfe7fe3186832bd).